tools/gulp/tasks/unit-test.ts

```diff
@@ -38,54 +35,3 @@ task('test:single-run', [':test:build'], (done: () => void) => {
     exitCode === 0 ? done() : process.exit(exitCode);
   }).start();
 });
-
-/**
- * [Watch task] Runs the unit tests, rebuilding and re-testing when sources change.
- * Does not inline resources.
- *
- * This task should be used when running unit tests locally.
- */
-task('test', [':test:build'], karmaWatchTask());
-
-/**
- * Runs a Karma server which will run the unit tests against any browser that connects to it.
- * This is identical to `gulp test`, however it won't launch and manage Chrome automatically,
- * which makes it convenient debugging unit tests against multiple different browsers.
- */
-task('test:static', [':test:build'], karmaWatchTask({browsers: []}));
```

Comment:
Could we leave this one in until we have a nice way of running Karma in Bazel without auto-watching? Otherwise it's very convoluted to debug failures on different browsers.

Reply:
You all should have the possibility to do this. Just running `bazel run {target}` opens the server on `localhost:9876/debug.html`. We should document this somewhere, but unless I miss something, this change should be good to go.

<sup>We need general documentation for bazel stuff I guess</sup>

Reply:
I meant debugging them on a browser that's not Chrome or Firefox. With `gulp test:static` it would start a Karma server without a browser and then you could connect to it to debug.

Reply:
But that's my point. It will bring up a server which you can access from _any_ browser. The tests will then run similarly to `gulp test:static`.

Reply:
That'll still execute the tests in both the original Chrome and the other browser you opened.

Reply:
I can't reproduce that behavior on my end. Browser tests are never executed unless a browser visits the `debug.html` page.

As I tried this with `ibazel` to test if it works with _watch mode_.. it looks like there is a ibazel bug causing the `stdin` to be closed (`Error writing success to stdin: write |1: The pipe is being closed.`).

I'll keep the `test:static` logic for now, but in most cases `bazel run` could be a replacement already (just mentioning as it can be quite helpful)

Reply:
@jelbourn @crisbeto Updated for now. I'll explore what the `ibazel` issue is. Other than that I don't see any other blockers.

@crisbeto maybe you can double-check or chat with me on what you mean with your last comment. I can't see that on my end. I see that the headless browser is instantiated.. but it never runs the test on that browser.